lib/ansible/cli/doc.py

```diff
@@ -100,6 +114,11 @@ def tty_ify(cls, text):
         t = cls._CONST.sub("`" + r"\1" + "'", t)        # C(word) => `word'
         t = cls._RULER.sub("\n{0}\n".format("-" * 13), t)   # HORIZONTALLINE => -------
 
+        t = cls._REFTAG.sub(r"", t)  # remove rst :ref:
+        t = cls._TERM.sub(r"", t)  # remove rst :term:
+        t = cls._NOTES.sub(r" note", t)  # nicer note
```

Comment:
How about
```suggestion
        t = cls._NOTES.sub(r" Note:", t)  # nicer note
```
instead?